src/helpers/index.tsx

```diff
@@ -8,9 +8,10 @@ import { ReactComponent as OhmImg } from "../assets/tokens/token_OHM.svg";
 import { ReactComponent as SOhmImg } from "../assets/tokens/token_sOHM.svg";
```

Comment:
What makes this file a tsx file?

Reply:
it holds React components: `SvgIcon`. If we move this elsewhere, it won't need to be.